runtime/doc/lsp.txt

```diff
@@ -1467,6 +1467,34 @@ progress_handler({_}, {_}, {params}, {client_id})
                 See also: ~
                     https://microsoft.github.io/language-server-protocol/specifications/specification-current/#workspace_executeCommand
 
+                                           *vim.lsp.handlers.signature_help()*
+signature_help({_}, {method}, {result}, {_}, {bufnr}, {config})
+                Parameters: ~
+                    {config}  table Configuration table.
+                              • border: (default=nil)
+                                • Add borders to the floating window
+                                • See |vim.api.nvim_open_win()|
+
+                See also: ~
+                    https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_declaration@seehttps://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_definition@seehttps://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_typeDefinition@seehttps://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_implementation|lsp-handler| for the method "textDocument/signatureHelp">
+
+                 vim.lsp.handlers["textDocument/signatureHelp"] = vim.lsp.with(
+                   vim.lsp.handlers.signature_help, {
+                     -- Use a sharp border with `FloatBorder` highlights
+                     border = {
```

Comment:
Not sure this is a good example as this can be achieved with just "single"

Reply:
Huh, this is an older doc comment, I'll change that though.

Reply:
This was auto generated from a PR I merged last night, we can change the example

Reply:
Maybe just use this branch and push to it with your changes, I initially just wanted to add the priority flag thing.

Reply:
I think we can just merge and fix the example later. @bfredl Do you have any suggestions on how you would like your border customization showcased? 😆

Reply:
you could just use "single" and reference the docs of nvim_open_win()

Reply:
@vigoux You can rebase and regenerate now